apps/hmc/hmc-hmi-outbound-adapter/perftest.yaml

```diff
@@ -5,6 +5,17 @@ metadata:
 spec:
   values:
     java:
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: '1536Mi'
```

Comment:
Based on current prod utilisation, I'm not sure this needs to be increased to 1536Mi. A more efficient configuration might be to leave at 1024 and increase the memory scaling threshold to 120%.

Reply:
this is the heavier service out of the 2, and can spike think leave as is for now